packages/mobile/e2e/scripts/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { E2E_TEST_WALLET } from './consts'
+export const Web3 = require('web3')
+export const ContractKit = require('@celo/contractkit')
+export const dotenv = require('dotenv')
+export const web3 = new Web3('https://alfajores-forno.celo-testnet.org')
+export const kit = ContractKit.newKitFromWeb3(web3)
+
+export const balanceError = async (address = E2E_TEST_WALLET, minBalance = 10) => {
```

Comment:
Following what I suggested from the check script:
```suggestion
export async function checkBalance(address, minBalance = 10) {
```

Also nit: Basic function better represents the intention to be called. Plus it's a few chars shorter to write.

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2325/commits/2a72d8204532a885b1fb40b95a45f1c7ce9f3fef!